src/components/Authentication/Login.vue

```diff
@@ -0,0 +1,104 @@
+<template lang="pug">
+.login-widget
+  v-alert.mt-0(
+      dismissible,
+      transition="scale-transition",
+      v-for="err in alerts.errors",
+      :key="err",
+      :value="!!err",
+      type="error") {{ err }}
+  v-container
+    v-form(@submit.prevent="login", ref="login")
+      v-text-field(
+          v-model="username",
+          label="Username or e-mail",
+          autofocus,
+          :rules="nonEmptyRules",
+          prepend-icon="$vuetify.icons.user",
+          type="text")
+      v-text-field(
+          v-model="password",
+          type="password",
+          label="Password",
+          :rules="nonEmptyRules",
+          prepend-icon="$vuetify.icons.lock")
+      v-layout(row)
+        v-btn.ml-0(type="submit",
+            color="primary",
+            :disabled="inProgress",
+            :loading="inProgress")
+          v-icon.aligned(left) $vuetify.icons.login
```

Comment:
Is the `aligned` class something from Vuetify? It doesn't seem to have any effect for me, but it's in a handful of places in these components.

Reply:
No, I added this because I didn't like the vertical alignment of icons in buttons.  They looked out of line with the text to me.  The actual styles are gone now. These were lingering and needed to be removed.